recipes/lsabgc/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "lsabgc" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Kalan-Lab/lsaBGC-Pan/archive/refs/tags/v{{ version }}.tar.gz 
+  sha256: 4030f51f1892be49908caf1814838384805eb5d37b7b9bc631f256cfae4d9bde 
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
+  run_exports:
+    - {{ pin_subpackage("lsabgc", max_pin="x") }}
+    
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python >=3.10
+    - r-base
+    - axel
+    - bioconda::zol >=1.4.11
+    - bioconda::orthofinder =2.5.5
+    - bioconda::pyseer >=1.3
+    - bioconda::gecco
+    - bioconda::prodigal
+    - bioconda::pyrodigal
+    - biopython
```

Comment:
Why some dependencies are prefixed with their channels and others not ?

Reply:
Thank you Martin,  I took out the channels since there is no overlap  with package names across them and will similarly do the same in my other repos in the future. Alternatively, if it is best practice to have channels for all the dependencies, please let me know and I can add them!

Reply:
I see channels used as a prefix very rarely in the recipes. That's why I ask what could be the benefit of them.
One positive thing having the channel/prefix might help the resolver to find the dependency faster, because AFAIK currently the build tool checks first in `conda-forge`, then locally and finally in `bioconda` (https://github.com/bioconda/bioconda-utils/blob/a5882cc87c0e451483d19f8bba701496cc058883/bioconda_utils/pkg_test.py#L98).